services/blockstorage/test/block_sync_test.go

```diff
@@ -196,8 +208,43 @@ func TestSyncCompletePetitionerSyncFlow(t *testing.T) {
 		harness.expectCommitStateDiffTimes(4)
 		harness.expectValidateWithConsensusAlgosTimes(4)
 
+		// fake the response
 		harness.blockStorage.HandleBlockSyncResponse(blockSyncResponse)
 
+		// verify that we committed the blocks
 		harness.verifyMocks(t, 4)
 	})
 }
+
+func TestSyncNeverStartsWhenBlocksAreCommitted(t *testing.T) {
```

Comment:
so how do I know what the sync timeout is? maybe it didn't start because the timeout is 100ms and you only waited 10ms?

Reply:
config, right? i never really fixed the manual setting of config values here :/ lets discuss tomorrow